server/spec/mutations/grid_certificates/request_certificate_spec.rb

```diff
@@ -106,6 +106,18 @@
       expect(authz.status).to eq :requested
     end
 
+    it 'adds error if acme server returns an error' do
+      expect(challenge).to receive(:request_verification).and_return(false)
```

Comment:
Just looking at the code, the [`Acme::Client::Resources::Challenges::Base#request_verification`](https://github.com/unixcharles/acme-client/blob/1b231de016a8df1ba2f4745f5bee612f73e7b2e4/lib/acme/client/resources/challenges/base.rb#L17-L20)  method returns a boolean from [`Faraday::Response#success?`](http://www.rubydoc.info/gems/faraday/0.7.4/Faraday/Response:success%3F), which is any non-2xx response.

Unfortunately no way to even get at the HTTP error response details via the `acme-client` API, I suspect? Although there's also some `Acme::Client::FaradayMiddleware#raise_on_error!` thing going on... TBH I don't know how this actually behaves...